### ISSUE TITLE
"Not exposing API key to production" comment makes no sense

### ISSUE THREAD

**Issue by danielo515** [locize/react-tutorial#1]
The below comment appears on several source files:

```
  apiKey: process.env.REACT_APP_LOCIZE_APIKEY, // YOU should not expose your apps API key to production!!!
```

This comment is misleading, confusing and probably wrong. You are literally bundling the app for production with the API keys and at the same time you state that this should not be done. It is clear that the client needs this API on the bundle, so there is no other option than to "expose" it, meaning it will be present in the production bundle.

Can you explain better what do you mean exactly?

**Comment by jamuhl** [comment]
The API key should not be exposed to production - and is also not needed. That API key is only needed if i18next saveMissing is true -> which should not be the case for production ----> every client request would trigger that missing request resulting in unneeded requests.

saveMissing should be done during development / testing / staging

**Comment by jamuhl** [comment]
https://github.com/locize/react-tutorial/tree/main/step_4 --> in the tutorial the API key is only bundled for development build not the production build

**Comment by adrai** [comment]
https://locize.com/blog/how-to-internationalize-react-i18next/#production

**Comment by danielo515** [comment]
Oh, ok, that makes much more sense.
Maybe you can rephrase it to something like `API key should only be used on development, not in production`.
The reason is that when I just read should not be "exposed" I thought it was a security risk, not something you only want to do on development.

Regards

**Comment by adrai** [comment]
It is indeed a security risk if you use the api-key on a publicly available production webapp.

**Comment by danielo515** [comment]
It is, but it is not like you should never let it get into the FE, because you have to provide it at least for the dev version
